md/release-notes.md

```diff
@@ -84,7 +84,23 @@ Bonita binaries as well as Studio installer are now officially signed on Windows
 ### Form mapping API error messages
 Improvements of messages in case of error on form mapping API calls have been done to explain the cause of error and give options for easier error recoveries.
 
-<a id="techonolgy-updates"/>
+### Performance (applies to Bonita 7.7.5+)
+The following improvements have been made to Bonita performance:
+- There is less I/O requests to temp folders
+- Work execution consumes less database resources
+- The login REST APIs mechanism consumes less resources
+- At engine startup and BPM services resume, the process dependencies loading consumes less memory
+
+A special attention has also been carried out to an improved deletion mechanism of archived cases. Deleting archived cases is now way more efficient.
```

Comment:
We should make it homogeneous with the chapter on 7.8.0.
Current proposition (waiting for review):
A few improvements have been brought to the overall Bonita efficiency:
- Deleting an archived case went down from seconds to milliseconds. Particularly enjoyable on environments with a large amount of archived data. Moreover, deleting cases in batches is now nearly as fast as deleting a single case and is not constrained by database connexion timeouts. 
- There is less I/O requests to temp folders
- Works execution uses less database resources
- The login REST APIs mechanism consumes less resources
- At engine startup and BPM services resume, the process dependencies loading requires less memory

Reply:
We plan to have the same content in 7.8, instead of the existing PR